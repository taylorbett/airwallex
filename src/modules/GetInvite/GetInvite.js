import React from 'react';

export class GetInvite extends React.Component {
    constructor(props) {
        super(props);

        this.state = {};
    }
    
    render() {
        return (
            <div className="get-invite">
                <h2>A better way to enjoy every day.</h2>
                <p>Be the first to know when we launch</p>
                <button onClick={() => this.props.handleModalOpen()}>Request an invite</button>
            </div>
        );
    }
}

export default GetInvite;