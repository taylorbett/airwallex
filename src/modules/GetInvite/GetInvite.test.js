import React from 'react';
import { GetInvite } from './GetInvite';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const registered = false;
    const handleModalOpen = () => {
        console.log('handleModalOpen');
    };
    const tree = renderer
        .create(
            <GetInvite
                registered={registered}
                handleModalOpen={handleModalOpen}
            />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});

