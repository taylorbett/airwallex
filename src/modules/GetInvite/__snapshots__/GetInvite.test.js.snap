// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="get-invite"
  style={
    Object {
      "padding": "60px 40px",
    }
  }
>
  <h2
    style={
      Object {
        "margin": "20px 0px",
      }
    }
  >
    A better way to enjoy every day.
  </h2>
  <p
    style={
      Object {
        "margin": "20px 0px",
      }
    }
  >
    Be the first to know when we launch
  </p>
  <button
    className="btn btn-primary"
    disabled={false}
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "border": "1px solid #fff",
        "borderRadius": "0px",
      }
    }
    type="button"
  >
    Request an invite
  </button>
</div>
`;
